Userland/Applications/Escalator/EscalatorWindow.cpp

```diff
@@ -1,27 +1,32 @@
 /*
```

Comment:
The commit subject doesn't capture at all what's going on. I suggest something like "Escalator: Enable forwarding stdin/stdout".

Reply:
But..that is just the same in different color? Allowing you to forward stdin/stdout is only one part of everything I did.

> Userland: Refactor Escalator
>
> - Escalator now uses `posix_spawn` instead of `exec`
> - Escalator is now able to forward stdin and stdout
> - A few general changes to improve code quality

I could try to split the commit up into multiple, but that will be a bit of work in this case.

Reply:
I agree that "Escalator: Enable forwarding stdin/stdout" isn't perfect, but at least it's better than "[Refactor](https://github.com/SerenityOS/serenity/blob/77f021f5f18300c2a1bf17128f20e02792b0d719/CONTRIBUTING.md?plain=1#L65)" :D

Reply:
> isn't perfect, but at least it's better

I do disagree. I will see if I can come up with something that works for both of us.

Reply:
"refactor" isn't a suitable word for a commit title, same as "fix". If you can't form a short description of the changes in the commit in the title, then perhaps it should be multiple commits, like you said.

Reply:
I think I found one that works.